### ISSUE TITLE
connection problems between fb page and website

### ISSUE THREAD

**Issue by rickymarso** [facebookincubator/Facebook-For-OpenCart#404]
hello I have problems with facebook application, it loads all the products but does not divide them by category, on the facebook page it shows the whole list of products not divided by category, am I wrong in something?  thanks

**Comment by dtying** [comment]
@rickymarso , yes you're right, currently the products are not grouped by collection.

**Comment by rickymarso** [comment]
so to divide them by category can I upload them only manually? thanks

**Comment by rickymarso** [comment]
Hi, can you make the change to the application, so as to view the categories as on the website? If so how much could it cost?

**Comment by dtying** [comment]
Hi @rickymarso, we're planning to enable collections sync feature in next half of the year, and it's still under discussion.
